### PR TITLE
Two more Glottocodes

### PR DESCRIPTION
**Pull request by chrzyki**
Does this make sense?

### CODE REVIEW

File: etc/languages.csv [hunk 1]
```diff
@@ -25,13 +25,13 @@ Dhayyi,dayi1244,Dayi
...
 Dhuwal,dhuw1249,Dhuwal
-Dhuwala,,
+Dhuwala,dhuw1248,Dhuwal-Dhuwala
 Diyari,dier1241,Dieri
 Djabugay,dyaa1242,Dyaabugay
 Djambarrpuyngu,djam1256,Djambarrpuyngu
 Djapu,djap1238,Djapu
 Djinang,djin1253,Djinang
-Durubul,,
+Durubul,yaga1256,Yagara-Jandai
```
Comment (xrotwang) [inline etc/languages.csv]: What's this assignment based on? I see one similar alternative name from ElCat for a doughter language of Yagara-Jandai. If that's the only evidence, shouldn't we assign to [yaga1262](https://glottolog.org/resource/languoid/id/yaga1262) - or better leave it unassigned?

Comment (chrzyki) [reply etc/languages.csv]: Assignment is a strong word, haha - I just had a quick look at https://www.slq.qld.gov.au/discover/aboriginal-and-torres-strait-islander-cultures-and-stories/languages/word-lists and compared some of the lexemes. Was less of an 'expert opinion' and more of an 'Hey experts, check this out, could this be?'. I agree, better leave it unassigned, but I thought the PR and asking doesn't do any harm. :)

**Comment by chrzyki** [comment]
More updates, thanks @natalia-morozova.

**Comment by chrzyki** [comment]
We both cross-referenced and check with the original data. Any feedback appreciated!

**Pull request merged**